resources/views/page/location/base.blade.php

```diff
@@ -0,0 +1,15 @@
+@extends('support::base')
+
+@section('content')
+    {{-- DO NOT REMOVE - page identity tag --}}
```

Comment:
Unrelated to this PR, but you could make this a partial which may better communicate the intent and avoid the comment.

Reply:
CC: @chx2 

Chris, could you make this partial?

Reply:
@drewroberts  The change to partials is already complete.  Ex - https://github.com/tipoff/locations/blob/faa8a3e67c6249629a34accc2b4035424c09bb39/resources/views/page/location/base.blade.php#L4